# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/ipc/SocketProcessParent.h
# Commit: a92c423736e5
# Full Hash: a92c423736e5979a8cf039017fd707d41f665b9c
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2020-05-25 15:42:48
# Regressor Bug: 1546358
# File Overlap Count: 1
# Description:
#   Bug 1546358 - Forward cache entry check to parent process r=dragana
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D75941
# ==============================================================================

diff -r b21cc935a336 -r a92c423736e5 netwerk/ipc/SocketProcessParent.h
--- a/netwerk/ipc/SocketProcessParent.h	Fri May 22 13:37:54 2020 +0000
+++ b/netwerk/ipc/SocketProcessParent.h	Mon May 25 11:07:07 2020 +0000
@@ -109,6 +109,10 @@
       PProxyConfigLookupParent* aActor, nsIURI* aURI,
       const uint32_t& aProxyResolveFlags) override;
 
+  mozilla::ipc::IPCResult RecvCachePushCheck(
+      nsIURI* aPushedURL, OriginAttributes&& aOriginAttributes,
+      nsCString&& aRequestString, CachePushCheckResolver&& aResolver);
+
  private:
   SocketProcessHost* mHost;
   UniquePtr<dom::MemoryReportRequestHost> mMemoryReportRequest;